### PR TITLE
added readme, not yet finalized, looking for ideas

### PR DESCRIPTION
**Pull request by Yoel-Perman-Brilliant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# SpikesLib - Examples
+Examples of how to use the [SpikesLib](https://github.com/Spikes-2212-Programming-Guild/SpikesLib2) library.
+
+## Includes:
+- Several subsystems and a drivetrain each with its own command examples.
+- Each subsystem has a RobotMap class, which holds the ports for the robot's devices.
```
Comment (tal-sitton) [inline README.md]: add the fact that they can just copy-paste the code, and only change the `RobotMap` for it to work

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# SpikesLib - Examples
+Examples of how to use the [SpikesLib](https://github.com/Spikes-2212-Programming-Guild/SpikesLib2) library.
+
+## Includes:
+- Several subsystems and a drivetrain each with its own command examples.
```
Comment (tal-sitton) [inline README.md]: Several subsystems and a drivetrain `,` each with its own command examples.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- Each subsystem has a RobotMap class, which holds the ports for the robot's devices.
```
Comment (tal-sitton) [inline README.md]: is `devices` the right word?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  - Each interface in RobotMap represents a different form of connection.
```
Comment (tal-sitton) [inline README.md]: is `form of connection` the right term?

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Yoel-Perman-Brilliant) [reply README.md]: probably not but nobody in here has got a better idea

Comment (tal-sitton) [reply README.md]: `a different port`

**Review by tal-sitton — APPROVED** [review]

**Pull request merged**